mcs/class/Mono.Profiler.Log/Mono.Profiler.Aot/ProfileBase.cs

```diff
@@ -0,0 +1,20 @@
+namespace Mono.Profiler.Aot {
+	public abstract class ProfileBase {
+
+		protected enum RecordType {
```

Comment:
We don't want to expose them externally, all these should be internal or private protected